test/jtreg-ext/requires/VMProps.java

```diff
@@ -333,6 +333,8 @@ protected void vmGC(SafeMap map) {
         map.put("vm.gc.ZSinglegen", () -> "" + (vmGCZ && (!genZ || genZIsDefault)));
     }
 
+    final String GC_PREFIX = "-XX:+Use";
+    final String GC_SUFFIX = "GC";
```

Comment:
Maybe move them to the beginning of the file and make them `private static final`? At least add a blank line after them and the following comment.

Reply:
I've moved them to the beginning of the file.